test/behaviour/concept/type/thingtype/ThingTypeSteps.java

```diff
@@ -133,6 +133,12 @@ public void thing_type_set_abstract(RootLabel rootLabel, String typeLabel, boole
         else get_thing_type(rootLabel, typeLabel).unsetAbstract();
     }
 
+    @When("{root_label}\\( ?{type_label} ?) set abstract: {bool}; throws exception")
+    public void thing_type_set_abstract_throws(RootLabel rootLabel, String typeLabel, boolean isAbstract) {
```

Comment:
a new step to used in the new scenarios to verify setting abstractness throws in some cases